tests/FunctionalTest.php

```diff
@@ -45,6 +45,12 @@ public function testBuildCssIfUsed(): void
 
         $asset = $assetMapper->getAsset('app.css');
         $this->assertInstanceOf(MappedAsset::class, $asset);
-        $this->assertStringContainsString('color: red', $asset->content);
+
+        if (null === $asset->content) {
+            // Starting with Symfony 6.4, the asset mapper only store the content during compilation.
+            $this->assertStringContainsString('color: red', file_get_contents($asset->sourcePath));
```

Comment:
I guess this check is not really of any value anymore (were basically testing the `file_get_contents` function), and probably could better skip this test in this case (only run for Symfony `6.3`?)

Reply:
Hm, since `$asset` is something that AssetMapper returns, I think this check makes sense as we checking for `$asset->sourcePath` valid property here, so fine to me